indigo_social/forms.py

```diff
@@ -63,6 +75,39 @@ def badge_choices():
     return sorted([(b.slug, '%s – %s' % (b.name, b.description)) for b in badges.registry.values() if b.can_award_manually])
 
 
+def process_profile_photo(profile_photo):
+    """ Process uploaded profile photo to meet specifications as follows:
+    - crop into a square if image is not
+    - resize to 200*200
+    - convert to jpeg
+    """
+    uploaded_photo = Image.open(profile_photo)
+    file_name = profile_photo.name.split('.')[0]
+    output = BytesIO()
+    width, height = uploaded_photo.size
+
+    if width != height:
+        new_width = min(width, height)
+
+        left = (width - new_width)/2
+        top = (height - new_width)/2
+        right = (width + new_width)/2
+        bottom = (height + new_width)/2
+        uploaded_photo = uploaded_photo.crop((left, top, right, bottom))
+        uploaded_photo = uploaded_photo.resize((200, 200), Image.ANTIALIAS)
+    else:
```

Comment:
line 97 is duplicated, you can drop the else and make line 99 always happen, and remove line 97

Reply:
fixed